circuits/balancesupdater.circom

```diff
@@ -29,47 +29,44 @@ template BalancesUpdater() {
     signal output update2;
     signal output fee2Charge; // amount of fee that needs to be discounted
 
+    signal output test;
```

Comment:
May be this is for test ?

Reply:
Yes, this signal was used for testing purposes and it should be removed

Reply:
nice !